tools/expotools/src/commands/UpdateVersionsEndpoint.ts

```diff
@@ -144,6 +183,8 @@ export default (program: Command) => {
     .option('-r, --release-note-url [string]', 'URL pointing to the release blog post.')
     .option('-k, --key [string]', 'A custom, dotted key that you want to set in the configuration.')
     .option('-v, --value [any]', 'Value for the custom key to be set in the configuration.')
-    .option('--delete', 'Deletes config entry under key specified by `--key` flag.')
+    .option('--delete', 'Deletes config entry under key specified by `--key` flag.', false)
+    .option('--delete-sdk', 'Deletes configuration for SDK specified by `--sdkVersion` flag.', false)
+    .option('--revert', 'Reverts changes on staging to the state from production.', false)
```

Comment:
Maybe something more like `pull`? idk, maybe not

Reply:
Revert implies that I can revert once, twice, thrice, and `pull` is more like a `update to some state` thing

Reply:
Maybe `reset`?